### PR TITLE
Remove fixed widths from navigation controls (fixes #1066)

### LINKED ISSUE #1066: Accessibility issue: when spacing is increased, Go button is pushed out of view and image string cut off
**UV version:** _universalviewer@4.0.25_

**I'm submitting a:** *bug report => please fork one of these [codesandbox examples](https://github.com/UniversalViewer/universalviewer/wiki/UV-Examples) with a repro of your issue and include a link to it below*

### Page area
header panel navigation bar

### Issue description
When the user adjusts the spacing on the site, this can break the header panel: 
- the term "image" before the number field gets cut off at the end and disappears behind the number field (see screenshot). This is worse for languages where this word has more letters 
- the Go button is pushed into a new line. This makes it mostly disappear, only the top is visible (see screenshot). 

### Steps to reproduce
1. install this bookmarklet which will increase the text spacing on the test page according to the WCAG requirements: https://dylanb.github.io/bookmarklets.html
2. open [this manifest](https://www.universalviewer.dev/#?xywh=583%2C1062%2C2492%2C1379&iiifManifestId=https%3A%2F%2Feap.bl.uk%2Farchive-file%2FEAP500-1-1%2Fmanifest&cv=)
3. switch to Welsh
4. click the bookmarklet and obsere the header panel: the term for "image" and the "Go" button

### Expected behaviour:
All components remain fully visible and legible when increased spacing is applied to the page.

### WCAG criterion
1.4.12 Text Spacing (Level AA 2.1 only)

### Related code:
"Image" string:

```html
<div class="mode"><label for="image">Delwedd</label><input type="radio" id="image" name="mode" tabindex="0" checked="checked" style="display: none;"><label for="page" class="disabled" style="display: none;">Tudalen</label><input type="radio" id="page" name="mode" tabindex="0" disabled="disabled" style="display: none;"></div>
```
"Go" button:
```html
<a class="go btn btn-primary" tabindex="0">GWELD</a>
```

### Rating:
high

### Other information:
**Note:** I have seen both those issues happening in Firefox even without spacing out the elements on the page when switched to French

To reproduce: Open the page in Firefox and switch to French. In this case both issues happen without even increasing the spacing on the page.

### Screenshot:
UV-cutoff-controls

### PR DESCRIPTION
**Pull request by LlGC-jop**
Fixes #1066 

Removed fixed widths from `.mode label` and `.search` the CY less file.

Changes fixed widths to be auto in the main pdf and paging headerpanel less files.

The auto widths seem fine in all languages at both normal and increased letter spacing, and at all screen widths where the panel appears.

### CODE REVIEW

**Review by demiankatz — APPROVED** [review]
This looks good to me, but I'll wait for one more review before merging! Thanks, @LlGC-jop!

**Comment by LanieOkorodudu** [comment]
> This looks good to me, but I'll wait for one more review before merging! Thanks, @LlGC-jop!

@demiankatz @LlGC-jop I've tested this fix in both Chrome and Firefox, as the issue was initially reported happening in Firefox. Everything looks good on my end, including when I switched the language to French. Thanks @LlGC-jop

**Comment by demiankatz** [comment]
Thanks, @LanieOkorodudu, I'll merge this now!

**Pull request merged**
